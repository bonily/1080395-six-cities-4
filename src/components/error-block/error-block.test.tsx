import * as React from "react";
import * as renderer from "react-test-renderer";
import ErrorBlock from "./error-block";

describe(`ErrorBlockSnapTest`, () => {
  it(`Errror Block should render block with error information`, () => {
    const tree = renderer
      .create(<ErrorBlock
      />
      ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
