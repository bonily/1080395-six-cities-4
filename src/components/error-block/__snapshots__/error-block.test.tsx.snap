// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorBlockSnapTest Errror Block should render block with error information 1`] = `
<div>
  <p>
    Sorry, something went wrong.
  </p>
  <p>
     Please, try again
  </p>
</div>
`;
